gcp/live/dev/infra/network/terraform.tfvars

```diff
@@ -23,5 +23,8 @@ create_static_ip_address = true
 # DNS zone names are dynamic (e.g. mrtyler.dev.gcp.gpii.net) so they will be
 # injected from outside. (If they're not, defaulting to no DNS records is
 # reasonable.)
-dns_zones = {}
+dns_zones   = {}
 dns_records = {}
+
+static_ip_region = "us-east1"
+cluster_subnets  = { "0" = "us-east1,10.16.0.0/20,10.17.0.0/16,10.18.0.0/16" }
```

Comment:
Since we use the region from `static_ip_region` with always the same set of subnets here (`10.16.0.0/20,10.17.0.0/16,10.18.0.0/16`), can we get rid of this and set it as default? Would save us repeating this multiple times.

Reply:
Sounds good!